.github/release_issue_template.md

```diff
@@ -24,5 +24,6 @@ Check with the docs squad coordinator for confirmation.
 
 - [ ] Build both SUMA and Uyuni docs from release branch and visually check output.
 - [ ] Publish to documentation.suse.com: https://github.com/uyuni-project/uyuni-docs/wiki/publishing-to-enterprise-endpoints
-- [ ] Publish to opensource.suse.com: https://github.com/uyuni-project/uyuni-docs/wiki/publishing-to-master-branches-on-github-pages
+- [ ] For Uyuni only - Build latest Uyuni docs with `make antora-uyuni`.
+- [ ] Create a new uyuni-YYYY-MM-pages branch based on the gh-pages branch(https://github.com/uyuni-project/uyuni-docs/tree/gh-pages). Add Julio as the Assignee if available so he can publish when ready.(Use a label with current Uyuni version (uyuni-2020.09)
```

Comment:
Do we need to mention that we only put the build dir in that branch? that had me scratching my head when i did it ;)

Reply:
Ah you are right. We should mentioned that we place the static content there. Also we need to make sure that **.nojekyll** is in the dir.